website/static/js/myProjects.js

```diff
@@ -554,6 +559,22 @@ var MyProjects = {
             self.breadcrumbs().push(linkObject);
         };
 
+        var sortCollections = function(collections) {
+            var collectionsToSort = [];
+            var bookmarksCollection = {};
+            for (var i=0; i<collections.length; i++) {
```

Comment:
You shouldn't need to loop through all the collections in order to remove the Bookmarks collection.

Also, since all the other items should already be sorted, you probably don't need to check `date_modified`. Maybe try this: https://stackoverflow.com/questions/23921683/javascript-move-an-item-of-an-array-to-the-front